.github/workflows/ci_eaa_kbc.yml

```diff
@@ -0,0 +1,57 @@
+name: image-rs eaa_kbc build
+on: [push, pull_request, create]
```

Comment:
This CI is testing the build for `signature` module with `attestation-agent/eaa_kbc` , we can use path filter to run when `signature` module is changed: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore